tests/Oryx.Integration.Tests/Node/NodeTestUsingZippedNodeModules.cs

```diff
@@ -164,5 +164,48 @@ public async Task BuildsAndRunsNodeApp_WhenPruneDevDependenciesIsTrue_AndNodeMod
                     Assert.Contains("Say It Again", data);
                 });
         }
+
+        [Fact]
+        public async Task CopiesNodeModulesInSubDirectory_ToDestination_WhenNodeModulesAreCompressed()
```

Comment:
Should we even test when compression is not used?

Reply:
good idea